arangod/Agency/Agent.cpp

```diff
@@ -2530,6 +2530,14 @@ void Agent::updateSomeConfigValues(velocypack::Slice data) {
     _config.setSupervisionDelayFailedFollower(u);
     _supervision->setDelayFailedFollower(u);
   }
+  slice = data.get("failedLeaderAddsFollower");
+  if (slice.isBool()) {
+    bool b = slice.getBool();
+    LOG_TOPIC("12345", DEBUG, Logger::SUPERVISION)
+        << "Updating failedLeaderAddsFollower to " << b;
+    _config.setSupervisionDelayFailedFollower(b);
```

Comment:
Shouldn't this be:
```suggestion
    _config.setSupervisionFailedLeaderAddsFollower(b);
```

Reply:
I hate C++. Why does that even compile ... yeah because you can implicitly convert ints to bools. :headdesk:

Reply:
good catch btw

Reply:
Oh dear. Done.